lib/commands/deploynodeapp.js

```diff
@@ -72,6 +78,35 @@ var descriptor = defaults.defaultDescriptor({
     name: 'Preserve policies from previous revision',
     shortOption: 'P',
     toggle: true
+  },
+  runtime: {
+    name: 'Runtime to deploy the Node.js app to',
+    shortOption: 'B',
+    required: false
+  },
+  stream: {
+    name: 'Stream EdgeServices API calls',
+    shortOption: 's',
+    toggle: true
+  },
+  'node-version': {
+    name: 'Version of node to use in EdgeServices',
+    shortOption: 'x',
```

Comment:
There are so many options already in use, i just started picking random ones...do an `apigeetool deploynodeapp --help` and itll list them, plus there are parent flags. Not really sure how to pick from the remaining.